04 Strategy Library/Fama French Five Factors/04 References.html

```diff
@@ -0,0 +1,5 @@
+<ul>
+    <li>
+      <a href="https://tevgeniou.github.io/EquityRiskFactors/bibliography/FiveFactor.pdf" rel="nofollow">A five-factor asset pricing model</a>
```

Comment:
Find a better reference, please. This is a transient link which might disappear.